config/environments/production.rb

```diff
@@ -74,4 +74,13 @@
   if AppConfig.environment.assets.host.present?
     config.action_controller.asset_host = AppConfig.environment.assets.host.get
   end
+  
+  config.dependency_loading = true if $rails_rake_task
+
+  require 'rack/rewrite'
+  config.middleware.insert_before(1, Rack::Rewrite) do
+    r301 '/apple-touch-icon-precomposed.png', 'https://s3.amazonaws.com/kickfinisher-prod/assets/bk-icon.png'            
```

Comment:
Line is too long. [121/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.